java.base/src/main/java/jdk/internal/misc/Unsafe$_patch.java

```diff
@@ -48,4 +48,11 @@ public int pageSize() {
         }
         return UnsafeConstants.PAGE_SIZE;
     }
+
+    @Replace
```

Comment:
should this also be geared with a `when = Build.Target.IsWasi.class` ?

Reply:
I remember @dmlloyd at some point said that it was always safe to patch out 🤔  but I may be wrong

Reply:
That seems plausible.